README.md

```diff
@@ -14,7 +14,13 @@ The static marketing site for login.gov
 
 ## Development
 
-This is a Jekyll-built static site. To start serve the site locally in development:
+This is a Jekyll-built static site. To install dependencies:
+
+```
+bundle install
+```
+
+To start serve the site locally in development:
```

Comment:
this is a pre-existing typo but as long as we're here,  can we update to say "to start serv**ing** the site" ?